pages/[network]/account/payments.tsx

```diff
@@ -11,49 +12,71 @@ import InternalLink from "components/internal-link";
 import NothingFound from "components/nothing-found";
 
 import { ApplicationContext } from "contexts/application";
+import { useAuthentication } from "contexts/authentication";
 
 import { formatNumberToCurrency } from "helpers/formatNumber";
 
+import { IPayment } from "interfaces/payments";
+
+import useApi from "x-hooks/use-api";
 import useNetworkTheme from "x-hooks/use-network";
 
-const PaymentItem = function () {
+interface IPaymentItem{
+  payment:  IPayment
+}
+
+const PaymentItem = function ({payment}:IPaymentItem) {
+  const { getURLWithNetwork } = useNetworkTheme()
+  const router = useRouter()
   return (
     <div className="bg-dark-gray px-3 py-2 d-flex justify-content-between mt-1 rounded-5">
       <div className="d-inline-flex row flex-shirk-1">
         <span className="caption-large text-uppercase text-primary mb-1">
-          {`${formatNumberToCurrency(12230304)} $BEPRO`}
+          {`${formatNumberToCurrency(payment?.ammount)} $BEPRO`}
```

Comment:
it might not be `$BEPRO` the currency name must be taken from the network information we are on

Reply:
@moshmage are we not going to use the transactional token now?

Reply:
> @moshmage are we not going to use the transactional token?

exactly: we should be showing the transactional token used for this bounty